TreeDB/docs/spec/typed-column-graph-search-prepared-views.md

```
@@ -30,6 +30,17 @@ authoritative typed-column and vector-index state assets
   -> optional final result-ID and document materialization
 ```
 
+#2043 closeout status: the combined prepared view is the admitted primary
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)